src/components/free-willy/LiquidationWithdrawals.tsx

```diff
@@ -1,343 +1,151 @@
-import { useCallback, useEffect, useState } from "react";
-import BigNumber from "bignumber.js";
+import { useConnectedWallet, useLCDClient, useWallet } from "@terra-money/wallet-provider";
+import { useAnchorLiquidationContract } from "hooks/useAnchorLiquidationContract";
+import { useEffect, useState } from "react";
 import {
-  CreateTxFailed,
-  Timeout,
-  TxFailed,
-  TxResult,
-  TxUnspecifiedError,
-  UserDenied,
+    CreateTxFailed,
+    Timeout,
+    TxFailed,
+    TxResult,
+    TxUnspecifiedError,
+    UserDenied,
 } from "@terra-money/wallet-provider";
-import useAddress from "hooks/useAddress";
-import { useRecoilValue } from "recoil";
-import { networkNameState } from "data/network";
 import 'components/free-willy/liquidation-withdrawals.css';
-import "style.css"
+import useNetwork from "hooks/useNetwork";
+import { Button, Dialog, DialogContent, DialogContentText, DialogTitle, Stack, Typography } from "@mui/material";
 
 export default function LiquidationWithdrawals() {
-  const lcd = useLCDClient();
-  const connectedWallet = useConnectedWallet();
-
-  var luna_bid_idx = Array<number>();
-  var eth_bid_idx = Array<number>();
-
-  const [luna_status, setLunaStatus] = useState<null | string>();
-  const [total_luna_bids, setLunaTotalBids] = useState<null | string>();
-  const [luna_bids, setLunaBids] = useState<null | Array<number>>();
-  const [pending_luna_collection, setLunaPendingCollection] = useState<
-    null | string
-  >();
-  const [luna_error, setLunaError] = useState<null | string>();
-  const [luna_tx_error, setLunaTxError] = useState<null | string>();
-  const [luna_tx_result, setLunaTxResult] = useState<null | TxResult>();
-
-  const [eth_status, setEthStatus] = useState<null | string>();
-  const [total_eth_bids, setEthTotalBids] = useState<null | string>();
-  const [eth_bids, setEthBids] = useState<null | Array<number>>();
-  const [pending_eth_collection, setEthPendingCollection] = useState<
-    null | string
-  >();
-  const [eth_error, setEthError] = useState<null | string>();
-  const [eth_tx_error, setEthTxError] = useState<null | string>();
-  const [eth_tx_result, setEthTxResult] = useState<null | TxResult>();
-
-  interface Bid {
-    idx: number;
-    collateral_token: string;
-    premium_slot: number;
-    bidder: string;
-    amount: number;
-    product_snapshot: number;
-    sum_snapshop: number;
-    pending_liquidated_collateral: number;
-    wait_end: number;
-    epoch_snapshot: number;
-    scale_snapshot: number;
-  }
-  interface BidsByUserResponse {
-    bids: Array<Bid>;
-  }
-
-  const withdrawLuna = useCallback(() => {
-    if (!connectedWallet) {
-      return;
+    const network = useNetwork();
+    const {getFilledBidsPendingClaimAmount, claimLiquidations} = useAnchorLiquidationContract(network.contracts.anchorLiquidation);
+    
+    const connectedWallet = useConnectedWallet();
+    
+    interface Bid {
```

Comment:
You can import this from the hook file
`import { useAnchorLiquidationContract, Bid } from "hooks/useAnchorLiquidationContract";`

No need to redeclare interfaces

Reply:
No need anymore, the Bid usage was removed with the latest update